### PR TITLE
Webb Relayer Testing Infrastructure 2.0

### LINKED ISSUE #54: [TASK] Update the Local Testnet with the new Signature Bridge and test it with the relayer

### LINKED ISSUE #102: [TASK] Indroduce E2E Testing Infrastructure Similar to the one for the DKG.
We need a new E2E Testing Infrastructure similar to the one we already got for the DKG w/ Signature Bridge.

This Infrastructure should allow us to do the following:

1. Create N-Sided Local EVM Testnets.
2. Run Multi-Node DKG System.
3. Run Multi-Relayers at the same time.
4. Test the Full System between the 3 components listed above.

### PR DESCRIPTION
**Pull request by shekohex**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [x] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: 
Closes #102 
Closes #54 

## What is the new behavior?

A clean start for the Relayer testing infrastructure

## Does this PR introduce a breaking change?
- [x] Yes
- [ ] No

All old tests have now gone, they are going to be implemented using the new infrastructure from this PR, in a cleaner way.

### CODE REVIEW

File: tests/evmTransactionRelayer.test.ts [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+      {
+        secretKey: PK2,
+        balance: ethers.utils.parseEther('1000').toHexString(),
+      },
+    ]);
+
+    wallet1 = new ethers.Wallet(PK1, localChain1.provider());
+    wallet2 = new ethers.Wallet(PK2, localChain2.provider());
+    // Deploy the token.
+    const localToken1 = await localChain2.deployToken(
```
Comment (saiakilesh) [inline tests/evmTransactionRelayer.test.ts:70]: Should this say `localChain2` or `localChain1`?

Comment (shekohex) [reply tests/evmTransactionRelayer.test.ts:50]: Thank you, Hawkeye 😉

File: tests/dummy.test.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+describe('Dummy Test', () => {
+  it('should pass', () => {
+    expect(true).toBe(true);
+  });
+});
```
Comment (dutterbutter) [inline tests/dummy.test.ts:5]: Do we need this / can this be removed?

**Review by dutterbutter — COMMENTED** [review]
Awesome job on this!

**Review by dutterbutter — APPROVED** [review]

**Review by saiakilesh — APPROVED** [review]
Nice work!

**Pull request merged**